app/forms/steps/income/income_before_tax_form.rb

```diff
@@ -0,0 +1,25 @@
+module Steps
+  module Income
+    class IncomeBeforeTaxForm < Steps::BaseFormObject
+      include Steps::HasOneAssociation
+      has_one_association :income_details, through: :applicant
+
+      # threshold being £12,475 currently
```

Comment:
Are we expecting this to change? Also shall we specify the time of the comment 'threshold being £12,475 as of Nov 2023'

Reply:
Was thinking we could tell from the commit but will amend as that would be clearer 👍